simple-chat-front/package.json

```diff
@@ -24,7 +25,8 @@
     "test:nowatch": "react-scripts test --watchAll=false",
     "eject": "react-scripts eject",
     "lint": "eslint src/",
-    "audit": "audit-ci --high"
+    "audit": "audit-ci --high",
+    "inspect": "jsinspect ./src"
```

Comment:
Modify the threshold to 20, you can check how [here](https://www.npmjs.com/package/jsinspect)